README.md

```diff
@@ -107,15 +107,15 @@ content <- client %>%
 # set an image for content
 
 content %>%
-  set_image_path("./my/local/image.png")
+  set_content_image("./my/local/image.png")
```

Comment:
Since you've renamed the other `$verb_image()` functions to `$verb_thumbnail()`, should this be `set_content_thumbnail()`?